### PR TITLE
Slope graph panel

### PR DESCRIPTION
**Pull request by KatrinaTurner**
a new panel plugin for a slope graph.

### CODE REVIEW

**Comment by KatrinaTurner** [comment]
my check is failing because it can't find the README.md, but its definitely there.

File: repo.json [hunk 1]
```diff
@@ -6021,6 +6021,23 @@
           }
         }
       ]
+    },
+    {
+      "id": "netsage-slope-graph-panel",
```
Comment (briangann) [inline repo.json:6026]: the linter won't accept this id, it would need to be renamed to "netsage-slopegraph-panel"

Comment (briangann) [reply repo.json:6026]: also you'll want to run the zip file through this: https://plugin-validator.grafana.net/

I did a quick check and it says the zip file doesn't contain a plugin.json, but the zip does have one under "src".

the zip package is expecting everything to be prebuilt and "live" at the top level of the extracted zip file, with "module.js" file and everything else needed.

**Comment by briangann** [comment]
i see what the problem is - the circleci config in your repo is not enough to get a plugin ready for publishing,  it would be better to switch to the github actions method and disable circleci.

**Comment by KatrinaTurner** [comment]
Ah, I see. Okay, thank you so much! 😄

**Pull request closed without merging**